get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
+	} catch ( e ) {
+		console.error(
+			'Could not read in the required configuration file.\n' +
+			'This file should exist as `config.js` inside the project root directory.\n' +
+			'Please consult the project README.md for further information.\n'
+		);
+		process.exit( 0 );
```

Comment:
I feel we should exit with an error so it will work correctly with other build tools. For instance running the following would confusingly boot up electron with either: 1) a broken React app or 2) the last succesfully built version excluding any changes you have made.

```
npm run build && electron .
```

Where as exiting with an error code would prevent `electron` from starting up.

Yes, `npm` is noisy in failure mode, but I think `npm` users have to put up with that.